omero/developers/Cpp.txt

```diff
@@ -25,11 +25,6 @@ Prerequisites
 Restrictions
 ------------
 
-Ice 3.4 will only build with GCC versions *older* than 4.6 (it
-contains broken headers which newer GCC versions will not parse). GCC
-4.4 is tested and recommended. This version will also fail to build
-with LLVM/clang++.
-
 Ice 3.5 will build with any GCC version up to 4.9, the latest stable
```

Comment:
As briefly discussed yesterday, we might want to remove this paragraph as well, maybe the whole section.  4.9 isn't the latest stable release any more, it's 5.3, so I would think we should update it or delete it.